CMakeLists.txt

```diff
@@ -74,6 +74,11 @@ target_include_directories(snmalloc_lib INTERFACE src/)
 
 if(NOT MSVC)
   find_package(Threads REQUIRED COMPONENTS snmalloc_lib)
+  if (SNMALLOC_ARM_ANDROID)
+    if (NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+      target_link_libraries(snmalloc_lib INTERFACE log c++_static c++abi m)
```

Comment:
with this I can build the library and run it correctly for armv7, but the size is larger.

Reply:
dynamic linking failed to resolve `__emutls_get_address`